app/views/topical_events/show.html.erb

```diff
@@ -123,30 +124,39 @@
 
   <div class="govuk-grid-column-two-thirds">
     <% if @topical_event.organisations %>
-          <div class="organisations">
-            <%= render "govuk_publishing_components/components/heading", {
-              text: t("topical_events.organisations"),
-              font_size: "l",
-              margin_bottom: 4,
-              padding: true,
-              border_top: 2,
-            } %>
-
-            <div class="govuk-grid-row">
-              <% @topical_event.organisations.each do |organisation| %>
-                <div class="govuk-grid-column-one-quarter govuk-!-padding-bottom-7">
-                  <%= render "govuk_publishing_components/components/organisation_logo", {
-                    organisation: {
-                      name: organisation[:title],
-                      url: organisation[:base_path],
-                      brand: organisation[:brand],
-                      crest: organisation[:crest],
-                    }
-                  } %>
-                </div>
-              <% end %>
+      <div class="organisations">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: t("topical_events.organisations"),
+          font_size: "l",
+          margin_bottom: 4,
+          padding: true,
+          border_top: 2,
+        } %>
+
+        <div class="govuk-grid-row">
+          <% @topical_event.organisations.each do |organisation| %>
+            <div class="govuk-grid-column-one-quarter govuk-!-padding-bottom-7">
+              <%= render "govuk_publishing_components/components/organisation_logo", {
+                organisation: {
+                  name: organisation[:title],
+                  url: organisation[:base_path],
+                  brand: organisation[:brand],
+                  crest: organisation[:crest],
+                }
+              } %>
```

Comment:
It looks like the only change here is the indentation.  Could this be corrected in the first commit, so there's no diff here?  That'll make it easier to see the changes.